api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -130,6 +131,9 @@ message OAuth2Config {
   // `RFC 6749 section 6 <https://datatracker.ietf.org/doc/html/rfc6749#section-6>`_), provided that the OAuth server supports that.
   // Default value is false.
   google.protobuf.BoolValue use_refresh_token = 12;
+
+  // The default lifetime in seconds of the access token, if omitted by the authorization server.
```

Comment:
Please add the default value if this field isn't set, and is not provided by the auth server.

Reply:
iiuc you mean add here info about the default so it renders in docs - i think its 0s

> If this value is not set it will default to 0s and require for the expiry to be set by the server

?

Reply:
updated on basis of my understanding

Reply:
ive also added a related tip in the sandbox https://github.com/envoyproxy/envoy/pull/31534/files#diff-e4e2790b4749c030d10ae18ac6c9358096e08b1c6758ceca190eff113a4d13e0R199-R214

ill update the main oauth docs once that has landed